09_Time_Series/Investor_Flow_of_Funds_US/Exercises_with_code_and_solutions.ipynb

```diff
@@ -1066,17 +1311,42 @@
     "### Step 10. Good, now we have the monthly data. Now change the frequency to year."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can change how you resample time period by changing \"rules\" options.  \n",
```

Comment:
I have added documentation link about how to use time period rules.
(I found the links through stack overflow, not from documentation of "resample" function. Official documentation should also add the links, I think.)